orc8r/gateway/go/services/magmad/service/magmad.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/aeden/traceroute"
 	"github.com/emakeev/snowflake"
 	"github.com/golang/glog"
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
```

Comment:
why are we introducing an unsupported package here

https://github.com/magma/magma/issues/12632

Reply:
The ticket #12632 was created as a result of realizing that the unsupported package was used here. I think this is for consistence but I agree that this PR should maybe also fix (parts of) #12632.

Reply:
@uri200 The package is removed in the final commit of this PR, so this is not part of the changes. (The PR isn't nicely tailored for commit by commit review, sorry.)

Reply:
> The ticket #12632 was created as a result of realizing that the unsupported package was used here. I think this is for consistence but I agree that this PR should maybe also fix (parts of) #12632.

Could you please elaborate? We introduce 12627 which we are not sure, we need and then we create #12632 to fix issues that #12627 will create?

Reply:
Oh,  sorry, that was not formulated clear enough.

#12632 was created while working on #12627. But it will also be solved here for the touched code. So no new instances of #12632 will be introduced and the code commented here is actually fixed in a later commit, as @alexzurbonsen explained.

Reply:
Cleaned up the commit history. Now all changes that replace merrors by hasicorp/go-multierror are in one commit.